src/components/AppSidebar/AppSidebar.vue

```diff
@@ -338,6 +342,9 @@ export default {
 		hasFigureClickListener() {
 			return this.$listeners['figure-click']
 		},
+		isCompact() {
+			return this.hasFigure && this.compact
```

Comment:
Okay, so, sorry about that, but I actually thought about this and I forgot one edge case.
One future addition is to go compact when scrolling down the sidebar. It will be cleaner for super long list like sharing can be.

Thus maybe leaving this to just the `compact` prop would be better?
What do you think?

Reply:
I don't mind, we can leave it to the prop.